src/openforms/scss/components/admin/_index.scss

```diff
@@ -21,6 +21,7 @@
 @import './logic-trigger';
 @import './logic-action';
 @import './logic-type-selection';
+@import './servicefetchconfiguration_form';
```

Comment:
can we stick to the existing file name style with dashes rather than underscores?